minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneClusterControl.java

```diff
@@ -150,7 +149,7 @@ public void adminStopAll() throws IOException {
    */
   public void setGoalState(String goalState) throws IOException {
     requireNonNull(goalState, "Goal state must not be null");
-    checkArgument(ManagerGoalState.valueOf(goalState) != null, "Unknown goal state: " + goalState);
+    ManagerGoalState.valueOf(goalState);
```

Comment:
updated errorprone flagged this is an impossible null check because Enum.valueOf does not return null, it throws an IllegalArgumentException.